onnx/backend/test/data/node/test_add_bcast/model.onnx

```diff
@@ -1,4 +1,4 @@
-backend-test:g
```

Comment:
Why are these models changed in this PR? 
`onnx/backend/test/data/node/test_add/model.onnx`
`onnx/backend/test/data/node/test_add_bcast/model.onnx`
This may be an artifact.

Reply:
I will revert it. Thanks.